### DISCUSSION TITLE
忽然看到今年更新了：请问什么时候可以做个mac版本 屏幕边缘利用滚轮方便且精准，是最有效率的交互

### DISCUSSION THREAD

**Discussion by constansino** [OneQuick/Feedback#98] [General]
屏幕边缘利用滚轮方便且精准，是最有效率的交互

**Comment by XUJINKAI** [comment]
目前没有做Mac版的打算，因为OneQuick使用了大量的Windows API，精力有限，想先把win上的搞好。

**Comment by constansino** [comment]
为什么中间弃坑了那么久啊，我可以通过源码自己学着做一个mac的吗 只要边缘滚动这个功能 不知道实现难度多大 不过好像bettermouse有这个功能

**Comment by XUJINKAI** [comment]
如果你是自用，配置都硬编码进去，应该也没几行代码。主要是需要通过系统API获取一些信息：例如鼠标滚轮的事件，鼠标位置，屏幕配置信息。如果不熟悉系统的话，调试这些API会麻烦一些。

**Reply by constansino** [reply]
好的谢谢佬，为什么中间弃坑了呀，欢迎回来 我们说不准能原创许多更有效率的交互

**Reply by XUJINKAI** [reply]
🤣生活所迫
